internal/ingress/metric/dummy.go

```diff
@@ -16,7 +16,15 @@ limitations under the License.
 
 package metric
 
-import "k8s.io/ingress-nginx/internal/ingress"
+import (
```

Comment:
things in this file weren't being used anywhere before right?

Reply:
correct